src/modules/brand/actions/listBrandItems.tsx

```diff
@@ -86,57 +97,70 @@ export const listBrandItems = createSmartAction(
               return response.data!;
             });
 
-            const poolsCopy = pools ? [...pools.list] : [];
-
-            return data
-              ?.map(item => {
-                const poolIndex = poolsCopy.findIndex(
-                  pool => pool.tokenId === item.id,
-                );
-                const pool = poolsCopy[poolIndex];
-
-                if (pool) {
-                  poolsCopy.splice(poolIndex, 1);
-                  return {
-                    ...item,
-                    supply: (() => {
-                      if (isEnglishAuction(pool)) {
-                        if (pool.state < AuctionState.NotSoldByReservePrice) {
-                          return pool.tokenAmount0;
-                        }
+            const poolsCopy = filterPools || [];
+
+            const getEnglishAuctionPrice = (pool: IEnglishAuctionDetails) => {
+              return pool.lastestBidAmount.isEqualTo(0)
+                ? pool.amountMin1
+                : pool.lastestBidAmount;
+            };
+
+            const getFixedSwapPrice = (pool: IFixedAuctionDetails) => {
+              return pool.price;
+            };
+
+            return (
+              data
+                ?.map(item => {
+                  const poolIndex = poolsCopy.findIndex(
+                    pool => pool.tokenId === item.id,
+                  );
+                  const pool = poolsCopy[poolIndex];
+
+                  if (pool) {
+                    poolsCopy.splice(poolIndex, 1);
+                    return {
+                      ...item,
+                      supply: (() => {
+                        if (isEnglishAuction(pool)) {
```

Comment:
it can so 
```
if (isEnglishAuction(pool) && pool.state < AuctionState.NotSoldByReservePrice) {
                        return pool.tokenAmount0;
                      }
                      if (!isEnglishAuction(pool) && pool.state < FixedSwapState.Canceled) {
                        return pool.quantity;
                      }
                      return 0
```